virttest/libvirt_vm.py

```diff
@@ -837,18 +866,39 @@ def add_kernel(help_text, cmdline, kernel_path=None, initrd_path=None,
         if smp:
             virt_install_cmd += add_smp(help_text, smp, vcpu_max_cpus,
                                         vcpu_sockets, vcpu_cores, vcpu_threads)
-
-        if params.get("numa", "no") == "yes":
+        numa = params.get("numa", "no") == "yes"
+        if numa:
             # Number of numa nodes required can be set in param
             numa_nodes = int(params.get("numa_nodes", 2))
             numa_vcpus = int(smp)
-            numa_memory = int(mem)
+            # virt-install takes --memory in MiB but --cpu cell adds numa
+            # memory in KiB by default
+            numa_memory = int(mem) * 1024
```

Comment:
@balamuruhans can we have this alone as separate PR as this might get fixed in virt-install and it will helpful to identify and revert later.